src/common/types.ts

```diff
@@ -1,9 +1,30 @@
 import { serverMessageKeys } from "symmetry-core"
-import { InlineCompletionItem, InlineCompletionList, Uri } from "vscode"
+import { InlineCompletionItem, InlineCompletionList, LogLevel, Uri } from "vscode"
 
 import { ALL_BRACKETS } from "./constants"
 import { CodeLanguageDetails } from "./languages"
 
+export const ErrorType = {
+  Default: 0, // default color
+  
+  Fetch_Error: 91, // red -- fetch error
+  Abort: 90,     // gray -- abort
+  Timeout: 33,   // yellow -- timeout
+};
+
+/**
+ * Log error messages in the console with error type and message listed in a specified color , and light red for error details
+ * @param errorKey - The key for the color in the colors object.
+ * @param message - The error message to display.
+ * @param error - The error detail to display in white, can be an Error object or a string.
+ */
+export function logConsoleError(errorKey: number, message: string, error: Error | string): void {
```

Comment:
Please could you move into the `Logger`  class and expose the method to where it needs to be used?